platforms/src/GtcStaking/Providers/communityStaking.ts

```diff
@@ -68,6 +70,38 @@ class CommunityStakingBaseProvider extends GtcStakingProvider {
       0
     );
   }
+
+  getCountRelevantStakesV2(communityStakes: StakeV2[], address: string): number {
```

Comment:
This looks great! The V2 endpoint is returning the data in a summary format already, so there's no need to do the totals for each address like we did on V1. So this could be simplified a bit. But it could also be left as-is!